src/Attachment/StorageManager.php

```diff
@@ -169,4 +169,28 @@ public function moveFile($old_path, $new_path)
     {
         return $this->mount_manager->move($old_path, $new_path);
     }
+
+   /**
+    * Check if File exists
+    * @param string $path
+    * @return bool
+    */
+    public function fileExists($path): bool
+    {
+         return $this->mount_manager->has($path);
+    }
+
+   /**
+    * Get Contents of file if it exists
+    * @param string $path
+    * @return string
+    */
+    public function getFileContents($path): string
+    {
+        if($this->mount_manager->has($path)){
+            return $this->mount_manager->read($path);
+        }else{
+            threw new League\Flysystem\FileNotFoundException($path);
```

Comment:
did you even test this code? `threw` is invalid keyword

Reply:
Never hit the error condition, so no. Will fix

Reply:
import namespace, use class name only